doc/operator-notes.md

```diff
@@ -62,3 +62,40 @@ If tooling is being used to generate Ignition configs, the tooling _should_ gene
 
 [selinux]: https://selinuxproject.org/page/Main_Page
 [restorecon]: https://linux.die.net/man/8/restorecon
+
+## Partition Reuse Semantics
+
+The `wipePartitionEntry` and `shouldExist` flags control what Ignition will do when it encounters an existing partition. `wipePartitionEntry` specifies whether Ignition is permitted to delete partition entries in the partition table.  `shouldExist` specifies whether a partition with that number should exist or not (it is invalid to specify a partition should not exist and specify its attributes, such as `size` or `label`).
+
+The following table shows the possible combinations of `shouldExist`, `wipePartitionEntry`, whether or not a partition with the specified number is present, and the action Ignition will take
```

Comment:
silly nit: I'd like the list of items here to follow the same ordering as the table headers